libbeat/processors/actions/docs/replace.asciidoc

```diff
@@ -11,15 +11,15 @@ contains a `field: field-name`, `pattern: regex-pattern` and
 `replacement: replacement-string`, where:
 
 * `field` is the original field name
-* `pattern` is regex pattern to match field's value
-* `replacement` is the replacement string to use for updating the field's value
+* `pattern` is the regex pattern to match the field's value
+* `replacement` is the replacement string to use to update the field's value
 
 The `replace` processor cannot be used to replace value with a completely new value. 
 
-TIP: You can replace field value to truncate part of field value or replace 
+TIP: Replace the field value to truncate part of the field value or replace 
 it with a new string. It can also be used for masking PII information.
```

Comment:
This reads really weird and I'm not quite sure what it should be saying. Replace the field value to truncate part of the value or replace it with a new string. It can also be used for masking PII information.

Is that better? Saying the field value twice didn't make sense.

Reply:
Wow, yeah, this is a really tough one to decipher. Here's what I think it's trying to say:

```suggestion
TIP: The `replacement` field value can be used to truncate the `field` value or replace
```